src/inc/Lfo.h

```diff
@@ -0,0 +1,119 @@
+#ifndef __Lfo_Hdr__
+#define __Lfo_Hdr__
+
+#include "RingBuffer.h"
+#include <cmath>
+
+class CWavetableLFO
+{
+public:
+    /*! Creates intance for the wavetable LFO
+    \param iWavetableLength wavetable length of the LFO
+    \param iSampleRate sample rate for the LFO
+    \param fFrequency frequency for the LFO in Hz
+    \param fAmplitude amplitude for the LFO
+    */
+    CWavetableLFO(int iWavetableLength, int iSampleRate, float fFrequency, float fAmplitude):
+        m_iSampleRate(iSampleRate),
+        m_dReadIdx(0)
+    {
+        double const l_dPi = 3.14159265358979323846;
```

Comment:
I was going to ask why not just use `M_PI`, but apparently that's actually a nonstandard extension (although it is part of [POSIX](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/math.h.html)). Weird!
Anyway, apparently C++20 [finally includes](https://en.cppreference.com/w/cpp/numeric/constants) a constant for `pi`, so maybe someday we can collectively stop rewriting this line of code. :stuck_out_tongue: